mods/persistence/controllers/subsystems/air.dm

```diff
@@ -0,0 +1,14 @@
+// TODO: Solve this issue properly
+// On load, we check turfs which have been marked for update and cull any which are outside. What's likely happening is they're being marked for update before
+// all z-levels are initialized, but it will take some time to track this down
+/datum/controller/subsystem/air/proc/cull_updated_turfs()
+
+	var/list/new_tiles_to_update = list()
+
+	var/list/curr_tiles = tiles_to_update
+	for(var/turf/T as anything in curr_tiles)
```

Comment:
Is "var/turf/T as anything in curr_tiles" better than just "var/turf/T in curr_tiles"?

Reply:
It is (or at least was previously) slightly faster by avoiding type checks